src/model/ActionSet.spec.cs

```diff
@@ -104,17 +158,21 @@ public void DeathCardTest()
             0,
             new List<Card> { },
             new List<Card> { },
-            new List<Card> { Card.LevCard(), Card.DimonCard(), Card.DimonCard() }
+            new List<Card> { Card.LevCard()}
           ) ,
           new Player(
             1,
             new List<Card> { },
             new List<Card> { },
-            new List<Card> { Card.DimonCard(), Card.DimonCard(), Card.DimonCard() }
+            new List<Card> { Card.DimonCard()}
           ) ,
         }
       );
 
+      var endTurnAction = ActionSet.PackAction(state, ActionType.EndTurn);
+      endTurnAction(state);
+      endTurnAction(state);
```

Comment:
I think we need to change the return value of `PackAction` to function without args.
Because it's already linked to `GameState`

Reply:
I don't understand what do you mean. Need to change return value of that PackAction 
`public static Func<GameState, GameState> PackAction(GameState state, ActionType type, int[] actualArgs = null, int[] remainArgs = null)
    {
      return PackAction(state, Actions[type], actualArgs, remainArgs);
    }`

Reply:
Yep.
`Func<GameState, GameState>` -> `Action<GameState>`

> P.S.
> You can wrap code like that:
> \`\`\` csharp
> code here
> \`\`\`

``` csharp
public static Func<GameState, GameState> PackAction(GameState state, ActionType type, int[] actualArgs = null, int[] remainArgs = null) { 
  return PackAction(state, Actions[type], actualArgs, remainArgs); 
}
```

Reply:
Need to edit each overload of `PackAction`

Reply:
Do we need to do it for both PackAction?

Reply:
> Do we need to do it for both PackAction?

Have u got notification about emotion set to your comment?